recipe/build.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# TELEMAC home directory
+export HOMETEL=$SRC_DIR/telemac-mascaret
+# Configuration file
+export SYSTELCFG=$HOMETEL/configs/systel.cfg
+
+# Configure PATH and PYTHONPATH
+export PATH=$HOMETEL/scripts/python3:$PATH
+export PYTHONPATH=$HOMETEL/scripts/python3 #:$PYTHONPATH
+
+rm -rf $HOMETEL/configs/*
+export MUMPSHOME=$PREFIX
+export SCALAPACKHOME=$PREFIX
+export BLACSHOME=$PREFIX
+export METISHOME=$PREFIX/lib
```

Comment:
Is this really needed ?

Reply:
no, I will change all instances to $PREFIX and test it

Reply:
actually changed all instances to $HOME. with:
HOME = PREFIX during host phase
HOME = HOMETEL during run phase

Reply:
I meant : Do we need to explicitly include these paths ?
bin, libs and includes are in PATH already no ?

On windows, I don't have to explicitly link metis, mpi, mumps has they are installed in correct folders.

Current systel.edf.cfg and others add these folders because vendors are not installed in usual directory (like /usr/bin, /usr/lib etc)
In Conda, if, you use packaged vendors (like you do), they are in correct directories and adding these paths should be optionnal.

Reply:
Let me try

Reply:
Confirming that everything works without including anything.

Reply:
Thanks - I would rather we try to avoid having so many extra environment variables - which are not useful.